legal-api/src/legal_api/reports/report.py

```diff
@@ -266,7 +266,10 @@ def _format_incorporation_data(self, filing):
         filing['listOfTranslations'] = filing['incorporationApplication'].get('nameTranslations', {})\
             .get('new', [])
         filing['offices'] = filing['incorporationApplication']['offices']
-        filing['shareClasses'] = filing['incorporationApplication']['shareClasses']
+        if filing['incorporationApplication'].get('shareClasses', None):
+            filing['shareClasses'] = filing['incorporationApplication']['shareClasses']
```

Comment:
This is to support existing incorporation applications which has the old structure in the filing json